Add deploying footnote for sapper build (#1366)
sveltejs/sapper: site/content/docs/09-building.md

node __sapper__/build
```

> If you're deploying your app to a different machine than the one you're building on, make sure that you install your production dependencies (required by `server.js`) by installing all dependencies from `package.json` using `--only=prod` (or `--only=production`):

Reviewer: We should remove this so as not to duplicate content from the deploying section

Author: Thanks for the reminder, I was still unsure back then but I think the wording in `11-deployment` states it more clearly, will remove this then. But before that, is it better if it was replaced with something like this?
> See [deployment section (link)] for instructions on deploying to a different machine

Reviewer: No, I don't think it's necessary and would leave it out

Author: Okay then, thanks for the review

node __sapper__/build
```

> If you're deploying your app outside your machine, make sure to satisfy your `server.js` by installing all your dependencies from `package.json` and to also include your `static` folder.

Reviewer 2: ```suggestion
If you're deploying your app outside your machine, make sure that you include production dependencies (required by) `server.js` by installing all dependencies from `package.json`:

 ```
   npm --production
 ```
 
   as well as including your `static` directory and it's contents.
```

not sure if we should also include `npm prune`, but I figure that we're not an npm tutorial.
